Classes/Form/Field/Text.php

```diff
@@ -15,6 +15,7 @@
  */
 class Text extends Input implements FieldInterface
 {
+    const DEFAULT_RTE_CONFIG = 'default';
```

Comment:
If we can find a constant in TYPO3 itself that has this default value, let's reference that instead of creating a new class constant here.